### PR TITLE
Fix typos etc

### PR DESCRIPTION
**Pull request by mattjdnv**
1. refs #142

### CODE REVIEW

File: fix_postgres.sh [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
+    max_files_per_process = 1000
+    work_mem = 16MB
+    maintenance_work_mem = 256MB
+    checkpoint_segments = 20
+    autovacuum = off
+    EOT
 
+    sudo service postgresql-$PG_VERSION restart
+    fi
+fi
```
Comment (sputnam) [inline fix_postgres.sh]: Github is complaining about not having a newline here.  Don't know if that's really an issue or not.

**Comment by mattjdnv** [comment]
I think the indent might have been the problem. Where did you see the GitHub error?

**Comment by sputnam** [comment]
screenshot from 2017-05-17 08 44 48

**Comment by mattjdnv** [comment]
Interesting. I have not seen that.  I'll add a newline.

**Comment by sputnam** [comment]
please build rpms

**Comment by sputnam** [comment]
please build rpms

**Comment by sputnam** [comment]
please build rpms

**Comment by sputnam** [comment]
please build rpms

**Comment by bmarchant** [comment]
@mattjdnv CONGRATULATIONS!!!

**Pull request merged**